ltm.go

```diff
@@ -959,7 +963,7 @@ func (b *BigIP) Monitors() ([]Monitor, error) {
 
 // CreateMonitor adds a new monitor to the BIG-IP system. <parent> must be one of "http", "https",
 // "icmp", or "gateway icmp".
-func (b *BigIP) CreateMonitor(name, parent string, interval, timeout int, send, receive string) error {
+func (b *BigIP) CreateMonitor(name, parent string, interval, timeout int, send, receive, user, pass string) error {
```

Comment:
We should try and avoid changing method signatures where possible to preserve compatibility. Since Go doesn't support overloading I would suggest a new create method that just takes a name + Monitor. e.g. `AddMonitor(name string, config *Monitor)`